dll/shellext/devcpux/lang/de-DE.rc

```diff
@@ -0,0 +1,41 @@
+LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL
+
+DLG_PROCESSORINFO DIALOGEX 0, 0, 252, 218
+STYLE DS_SHELLFONT | WS_POPUP | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME
+CAPTION "CPU"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "Merkmale:", -1, 7, 23, 50, 8
+    LTEXT "test", IDC_FEATURES, 57, 23, 181, 8
```

Comment:
"test"?

Reply:
Ja. ist in den anderen Übersetzungen auch so, daher hab ich es so gelassen.